_data/health.yml

```diff
@@ -153,6 +153,16 @@ websites:
     regions:
       - us
 
+  - name: MyChart by Epic
```

Comment:
```suggestion
  - name: MyChart
```

Reply:
Epic is the company providing MyChart backend product, you can see it is included in the logo on the website. For people searching for setup / clarification I think that the full name may be more helpful to find it easily

Reply:
We usually do not include the company name if the product name is distinctive enough. People will probably search for "MyChart" rather than "Epic", I suppose.